### PR TITLE
Fix #581, Broken links in docs

### LINKED ISSUE #581: Broken links in docs
**Checklist (Please check before submitting)**

* [x] I reviewed the [Contributing Guide](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md).
* [x] I performed a cursory search to see if the bug report is relevant, not redundant, nor in conflict with other tickets.

**Describe the bug**
Just noticed a few broken links when perusing CONTRIBUTING.md and SECURITY.md - seems like some links were broken in the reorganization of the last commit 0e597817b785bbc7fd84c8c08c41b40bc2aedcd1

e.g. https://github.com/nasa/cFS/blob/7281299bf45d089b74f5e8e95c26ebd2ec8b5f62/CONTRIBUTING.md?plain=1#L60

**Expected behavior**
Links (internal + external) should work correctly to improve usability.

**System(s) tested on**
n/a

**Reporter Info**
@thnkslprpt

### PR DESCRIPTION
**Pull request by thnkslprpt**
**Checklist (Please check before submitting)**

* [x] I reviewed the [Contributing Guide](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md).
* [x] I signed and emailed the appropriate [Contributor License Agreement](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) to GSFC-SoftwareRelease@mail.nasa.gov and copied cfs-program@lists.nasa.gov.

**Describe the contribution**
Fixes #581
Corrects a few broken links in the contributing guide.
Also cleaned up a few other links for consistency.

**Testing performed**
Tested the links - working now.

**Expected behavior changes**
Links will work.

**System(s) tested on**
Chrome browser.

**Contributor Info - All information REQUIRED for consideration of pull request**
@thnkslprpt

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -132,21 +132,21 @@ For issues related to continuous testing workflows or multi-component "epics" st
...
+- [CF](https://github.com/nasa/CF/issues)
+- [CS](https://github.com/nasa/CS/issues)
+- [DS](https://github.com/nasa/DS/issues)
+- [FM](https://github.com/nasa/FM/issues)
+- [HK](https://github.com/nasa/HK/issues)
+- [HS](https://github.com/nasa/HS/issues)
+- [LC](https://github.com/nasa/LC/issues)
+- [MD](https://github.com/nasa/MD/issues)
+- [MM](https://github.com/nasa/MM/issues)
 - [SC](https://github.com/nasa/SC/issues)
```
Comment (chillfig) [inline CONTRIBUTING.md:151]: Consider adding SCH, git commit --amend, and git push --force

- [SCH](https://github.com/nasa/SCH//issues)

**Review by chillfig — CHANGES_REQUESTED** [review]
SCH can also be added to line 171.

**Comment by chillfig** [comment]
Hi @thnkslprpt , thank you for the pull request. It looks good! Just some comments about SCH.

File: CONTRIBUTING.md (hunk 1 above)
Comment (thnkslprpt) [reply CONTRIBUTING.md:152]: Added.

**Comment by thnkslprpt** [comment]
> Hi @thnkslprpt , thank you for the pull request. It looks good! Just some comments about SCH.

No worries mate.
That's added now.

**Review by chillfig — CHANGES_REQUESTED** [review]
We want to follow a commit message convention throughout all cFS. At least, we want to change the commit message: "Add SCH to list of apps". Would you please squash the commits? `git rebase -i HEAD~3` from your local cFS repo, squash the latter 2 commits, reword the 1 commit to "Fix #581, Updates broken links in docs". I think this makes sense since most of this pr's work is in lieu with that message. What do you think?

**Comment by thnkslprpt** [comment]
> What do you think?
Yeah makes sense. Let me know if I've updated it successfully now Justin.
Cheers

**Review by chillfig — APPROVED** [review]

**Pull request merged**